### DISCUSSION TITLE
After install OpenShellSetup_4_4_194.exe on Win 10 can't dock taskbar to right side, like before

### DISCUSSION THREAD

**Discussion by kevinlkoehler** [Open-Shell/Open-Shell-Menu#2076] [Q&A]
After install OpenShellSetup_4_4_194.exe on win10 can't dock taskbar to right side, like before, after install reboot was required, and taskbar was on left-side, tried dragging it  but then it stuck to the screen bottom - is there a short-cut or option to shift it to the right side again?
taskbar is unlocked - or maybe a tweak tool.

**Comment by ge0rdi** [comment]
For me it works without a problem.
I can drag taskbar to whatever screen edge I want.

Make sure you don't have taskbar _locked_ in order to move it.
Right click on taskbar and untick `Lock the taskbar` option.

**Reply by kevinlkoehler** [reply]
Thanks, still looking to reset it. If there was a config.ini or reg setting I could change.
